.ops/ecamp3-logging/README.md

```diff
@@ -5,12 +5,7 @@ ecamp3 is running.
 
 ## Prerequisites
 
-You need to add the fluent helm repository:
-
-```shell
-helm repo add fluent https://fluent.github.io/helm-charts 
-helm repo update
-```
+You need the helmfile in addition to kubectl and helm.
```

Comment:
Instructions or a link on how to install the helmfile command might be useful. But maybe the wiki is a better place for that.

Reply:
I can link the action in the wiki as soon as this is merged:
https://github.com/ecamp/ecamp3/pull/7879/files/16d5d30aa3235f26f0be97ad800d2a23d6c78800#diff-a6c41caf93cca9a01ad670187f58916de37441dec13c5dfb1b91252a7885c947